ukat/vessels/phase_contrast.py

```diff
@@ -0,0 +1,288 @@
+    std_velocity : np.ndarray
+        List containing the std dev of the velocity values (cm/s) per phase.
+    rbf : np.ndarray
+        List containing the Renal Blood Flow values (ml/min) per phase.
+    mean_velocity_global : float
+        Average velocity (cm/s) accross the different phases.
+    mean_rbf : float
+        Average Renal Blood Flow (ml/min) accross the different phases.
+    resistive_index : float
+        A prognostic marker in renal vascular diseases which range is [0, 1].
+    """
+
+    def __init__(self, velocity_array, affine, mask=None):
+        """Initialise a PhaseContrast class instance.
+
+        Parameters
+        ----------
+        velocity_array : np.ndarray
+            A 3D array containing the velocity images of the phase contrast
+            sequence, i.e. the dimensions of the array are [x, y, p], where p
+            corresponds to the phase (or trigger delay).
+        affine : np.ndarray
+            A matrix giving the relationship between voxel coordinates and
+            world coordinates.
+        mask : np.ndarray, optional
+            A boolean mask of the voxels to fit. Should be the shape of the
+            the raw data.
+        """
+        self.shape = velocity_array.shape
+        self.affine = affine
+        self.pixel_spacing = (np.linalg.norm(self.affine[:3, 1]),
+                              np.linalg.norm(self.affine[:3, 0]))
+        # Generate a mask if there isn't one specified
+        if mask is None:
+            self.mask = np.ones(self.shape, dtype=bool)
+        else:
+            # Set masked areas to np.nan to enable differentiation between
+            # masked voxels and voxels where data is zero using np.nanmean.
+            self.mask = np.where(mask, mask, np.nan)
+        self.velocity_array = np.abs(velocity_array * self.mask)
+        self.num_pixels = []
+        self.area = []
+        self.min_velocity = []
+        self.mean_velocity = []
+        self.max_velocity = []
+        self.std_velocity = []
+        self.rbf = []
+        self.mean_velocity_global = 0
+        self.mean_rbf = 0
+        self.resistive_index = 0
+        if len(self.shape) == 3:
+            # Extract number pixels, area, velocity stats (min, mean, max, std)
+            # and renal blood flow (RBF)
+            self.num_pixels = np.count_nonzero(~np.isnan(self.velocity_array),
+                                               axis=(0, 1))
+            # area = (num_pixels * mm * mm * 0.01) = cm2
+            self.area = (self.num_pixels * self.pixel_spacing[0] *
+                         self.pixel_spacing[1] * 0.01)
+            self.min_velocity = np.nanmin(self.velocity_array, axis=(0, 1))
+            self.mean_velocity = np.nanmean(self.velocity_array, axis=(0, 1))
+            self.max_velocity = np.nanmax(self.velocity_array, axis=(0, 1))
+            self.std_velocity = np.nanstd(self.velocity_array, axis=(0, 1))
+            # q = (60s * cm2 * cm/s) = cm3/min = ml/min
```

Comment:
You're just multiplying it by a constant of 60 rather than 60 sec.

`# q = (60 * cm2 * cm/s) = ml/min`